_sass/blocks/_revenue.scss

```diff
@@ -4,19 +4,180 @@
 //
 // Styleguide blocks.revenue
 
-section.revenue {
+.revenue-section {
+  $dark-blue: #4b9bbf;
+  $positive-fill: $pale-blue;
+  $negative-fill: #bbb;
+
+  .filters {
+    color: $dark-blue;
+  }
+
+  button.toggle {
+    background: transparent;
+    // border: 1px solid;
+    color: $dark-blue;
+    display: block;
+    font: inherit;
+    margin: 0 auto;
+    padding: 0;
+    position: relative;
+    text-align: left;
+    width: 100%;
+
+    &:after {
+      content: $fa-var-chevron-down;
+      font-family: FontAwesome;
+      font-size: 1.2em;
+      line-height: 1;
+      margin-top: -0.5em;
+      position: absolute;
+      right: 0.5em;
+      top: 50%;
+    }
+
+    &[aria-expanded=true] {
+      margin-top: 1em;
+    }
+
+    &[aria-expanded=true]:after {
+      content: $fa-var-chevron-up;
+    }
+  }
+
+  [aria-expanded=false] {
+    display: none;
+  }
+
+  div[class^=slab] {
+    padding-bottom: 1em;
+    padding-top: 1em;
+  }
+
+  eiti-slider {
+    margin-top: 1em;
+  }
 
   svg[is='eiti-map'] {
+    display: block;
     height: 250px;
+    margin-bottom: 1em;
     max-height: 30vh; // 30% of the viewport height
+    width: 100%;
   }
 
   .regions .region {
-    // display: none;
+    display: none;
 
     &.active {
       display: block;
     }
   }
 
+  .map-legend {
+    display: flex;
+    margin-bottom: 1em;
+
+    @include respond-to(medium-down) {
+      font-size: 0.8em;
+    }
+
+    .step {
+      display: block;
+      flex: 1;
+      border-bottom-style: solid;
+      border-bottom-width: 0.5em;
+      padding-left: 0.1em;
+    }
+
+    &.legend-invalid {
+      .step .label {
+        visibility: hidden;
+      }
+    }
+  }
+
+  .timeline {
+    shape-rendering: crispEdges;
+
+    line {
+      fill: none;
+      stroke: #4b9bbf;
```

Comment:
Color literals like `#4b9bbf` should only be used in variable declarations; they should be referred to via variable everywhere else.